src/models/userDao.js

```diff
@@ -0,0 +1,35 @@
+const appDataSource = require('./dataSource')
+
+const signUp = async (kakaoId, email, name, profileImage) => {
+    const result = await appDataSource.query(`
+     
+    INSERT INTO users(
+        kakao_id,
+		email,
+        nickname,
+        profile_image
+    ) VALUES (?, ?, ?, ?)`
+    , [kakaoId, email, name, profileImage]
+    )
+    return result
+}
+
+const getUserById = async kakaoId => {
+    const [user] = await appDataSource.query(`
+    SELECT
+        kakao_id,
+        email,
+        nickname,
+        profile_image
+    FROM users u
+    WHERE kakao_id=?`
+    , [kakaoId]
+    )
+    return user
+} 
```

Comment:
토큰을 발행할 때 주입하는 payload에 user의 id 값을 주입한 이후, 해당 함수에서도 user의 id 값을 통해 데이터를 조회할 수 있도록 수정해주세요.

user의 id도 반환하는 것이 좋을 것 같습니다!

```suggestion
const getUserById = async (userId) => {
    const [user] = await appDataSource.query(`
    SELECT
        id,
        kakao_id,
        email,
        nickname,
        profile_image
    FROM users
    WHERE id=?`
    , [userId]
    )
    return user
} 
```

Reply:
반영 완료하였습니다.